src/main/java/racingcar/RacingManager.java

```diff
@@ -0,0 +1,59 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Iterator;
+
+public class RacingManager {
+    RacingSetting setting = new RacingSetting();
```

Comment:
```suggestion
    RacingSetting setting = new RacingSetting();
    List<Car> cars = setting.createCar();
    int racingCount = setting.getRacingCount();
```
RacingSetting.class의 racingSetting()에 적은 이유와 같습니다!